modules/serverless-traffic-routing-examples.adoc

```diff
@@ -0,0 +1,76 @@
+[id="serverless-traffic-routing-examples_{context}"]
+= Traffic routing examples
+
+When you create a Knative service, it does not have any default `traffic` spec settings. By setting the `traffic` spec, you can split traffic over any number of fixed revisions, or send traffic to the latest revision.
+
+.Splitting traffic routing between multiple revisions
```

Comment:
The block title syntax (`.Block title`) is generally reserved for code blocks, tables, and figures.

Can you revise to use a heading?
* `== Split traffic routing between multiple revisions`
* Add an ID attribute that includes `_{context}`, or precede the heading with `[discrete]` if your goal is to prevent them from appearing in the page TOC. Info about discrete is in the doc guidelines.

I'm tempted to suggest repetition of the "== Example: ..." pattern. If you do, consider changing the H1 in this file to "Knative CLI traffic management|splitting examples" and see if the page TOC matches the experience you want to create for customers.